data/json/items/migration.json

```diff
@@ -149,5 +149,11 @@
     "type": "MIGRATION",
     "replace": "jar_glass_sealed",
     "contents": [ "tomato_canned" ]
+  },
+  {
+    "id": "jar_kompot",
```

Comment:
OK, did it.
I have learned a bit of `jq`, but more importantly, about the futility of non-automated sorting.

> Clean and well organised JSON is as important as good C++ code and we have historically done very badly at the former.

Requiring manual cleanups for something as brainless as sorting order is to organization what premature optimization is to performance.

It's fine to require alphabetic order when adding 1-3 new entries, but when adding 10+, the sorting requirement is by an order of magnitude more likely to cause merge conflicts than inserting them all in a single spot.